GITHUB_ORG_MANAGEMENT_POLICY.md

```diff
@@ -65,13 +65,29 @@ rejects the request, then the request is denied.
 In certain cases, OpenJS Cross Project Council and/or OpenJS Foundation Board
 of Directors approval may also be required.
 
-### Teams
+### Repository Team Ownership
 
 When making a request to create a new repository, specify the team(s) that will
 have write or admin access. If there is not an appropriate team to maintain a
 new repository, request a new team. Approval is automatic if there are no
 objections from TSC or CommComm after 72 hours.
 
+## Teams
+
+To join a Working Group team or a team who fall under the charter of a Working 
+Group, read that Working Group governance documentation. For other teams, open
+an issue on the [Node.js admin repository][nodejs/admin].
```

Comment:
This isn't how CommComm does things afaik - generally members are PR-ed in and then the Initiatives (which is a group set left out here) request that the only member of CommComm who is an org owner (the chairperson) add them 😬

Reply:
That's also how most working groups work AFAIK. The policy/guidelines proposed here should only apply to teams who are not tied to a repository.

Reply:
Ah my bad. Failure of my reading. Would it be possible to clarify that in more specific language in `For other teams, [...]` ?

Additionally, it would be nice if `To join a Working Group team or a team` also included `Initiatives`.